folding/src/quadraticization.rs

```diff
@@ -90,11 +90,23 @@ fn lower_degree_to_1<C: FoldingConfig>(
     let degree = exp.degree();
     match degree {
         1 => exp,
-        _ => {
-            let exp = lower_degree_to_2(exp, rec);
-            let id = rec.get_id(exp);
-            FoldingExp::Atom(ExtendedFoldingColumn::WitnessExtended(id))
-        }
+        _ => match exp {
+            FoldingExp::Add(e1, e2) => FoldingExp::Add(
```

Comment:
Do you have any example testing the output of quadraticization of simple expressions, like `X1 * X2`, `X1 * X2 * (X3 + X4)`, etc?